tests/test_mirrors.py

```diff
@@ -60,16 +59,48 @@ def setUp(self):
      'mirror3': {'url_prefix' : 'http://mirror3.com',
                  'targets_path' : 'targets',
                  'confined_target_dirs' : ['targets/release/v2/']},
+     # confined_target_dirs = [] means that none of the targets on
+     # that mirror is available.
```

Comment:
Decided to leave a comment about this, because I made a mistake during testing connected with how we interpreted `confined_target_dirs = []`.

Reply:
Thanks for adding the comment. I wonder if we should document this better throughout the code, most notably in updater.py. AFAICT if `confined_target_dirs` is a `None` value the mirror will never be considered/used. Does that match your understanding?

Reply:
Yes, I checking if `confined_target_dirs` is a falsely value like that:

```
confined_target_dirs = mirror_info.get('confined_target_dirs')
# confined_target_dirs is an optional field
if confined_target_dirs and not in_confined_directory(full_filepath,
          confined_target_dirs):
     ...
```

I don't want us to leave a lot of comments (especially publicly) about what `confined_target_dirs = []` means, because I prefer if we advertise to not use the field if you don't want confinement for your mirrors. 

This unit test is the only place where `confined_target_dirs = []` could be seen and that's why it's worth leaving that comment.